torch/_subclasses/functional_tensor.py

```diff
@@ -64,6 +64,14 @@ class FunctionalTensor(torch.Tensor):
         torch.ops.prim.device.default,  # type: ignore[has-type]
     ]
 
+    # These are ops that claim to be functional, but actually are not
+    fake_functional_ops = [
+        torch.ops.aten.dropout.default,  # type: ignore[has-type]
+        torch.ops.aten.batch_norm.default,  # type: ignore[has-type]
+        torch.ops.aten.native_batch_norm.default,  # type: ignore[has-type]
+        torch.ops.aten._batch_norm_impl_index.default,  # type: ignore[has-type]
```

Comment:
Hmm, I think we want every op that is a "maybe-mutating/maybe-aliasing" op to be in this list. That also includes:
```
aten.reshape
aten.contiguous
aten.cudnn_batch_norm
aten.miopen_batch_norm
...??? (hopefully that's it)
```

Reply:
Can you also add a test, similar to the batch norm test that you have (proving that batch_norm still decomposes with pre-dispatch functionalization), but also for:

reshape() -> should decompose into view()

contiguous -> should be a no-op if the input is already contiguous

Reply:
I think reshape and contiguous are already marked as view op. so i think we should be good without adding it to the list. Wrote some test cases to verify it